lib/components/Snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactNode, useEffect, HTMLAttributes, useRef } from "react";
+import { Text } from "../Typography";
+import clsx from "clsx";
+import { Button } from "../Button";
+import { LabelPairedXmarkSmBoldIcon } from "@deriv/quill-icons";
+import { useSnackbar } from "@hooks/useSnackbar";
+
+export interface SnackbarProps extends HTMLAttributes<HTMLDivElement> {
+    icon?: ReactNode;
+    id: string;
+    isShown: boolean;
```

Comment:
I would like to understand if this props is acting as a flag to check whether the snackbar has been shown already otherwise, I think we can use `isVisible` instead.

Reply:
i put "isShown" because provider maintains, manages, and controls whether or not a snackbar in the queue is shown already or not. 

I can keep isShown on provider side and isVIsible on component side.

What do you think?